extensions/stats/testdata/client.yaml

```diff
@@ -84,14 +84,14 @@ static_resources:
                 route: 
                   cluster: server
           http_filters:
-          - name: envoy.wasm
```

Comment:
Perhaps I missed it, but change to server.yaml?

Reply:
The test config is no longer functional. We can remove it. Filter state is not easily accessible to native filters, which is its downside.